src/Microsoft.Identity.Client/Features/TokenCache/TokenCacheExtensions.cs

```diff
@@ -73,13 +75,15 @@ public static void SetBeforeWrite(this TokenCache tokencache, TokenCache.TokenCa
         /// <param name="state"></param>
         public static void Deserialize(this TokenCache tokenCache, byte[] state)
         {
+            RequestContext requestContext = new RequestContext(Guid.Empty);
+
             lock (tokenCache.LockObject)
             {
                 Dictionary<string, IEnumerable<string>> cacheDict = JsonHelper
                     .DeserializeFromJson<Dictionary<string, IEnumerable<string>>>(state);
                 if (cacheDict == null || cacheDict.Count == 0)
                 {
-                    //TODO log about empty cache
+                    requestContext.Logger.Info(string.Format(CultureInfo.InvariantCulture, "The cache is empty"));
```

Comment:
I don't think you need to call string.Format() if the string is not inserting variables.  Just pass the string without the call to string.Format() and CultureInfo.